_pytest/main.py

```diff
@@ -648,8 +648,37 @@ def _recurse(self, path):
         ihook.pytest_collect_directory(path=path, parent=self)
         return True
 
+    def _locate_module(self, modulename, searchpath):
+        """Find the locations of a module or package in the filesytem.
+
+        In case of a presumptive namespace package return all of its possible
+        locations.
+
+        """
+        try:
+            fd, pathname, type_ = imp.find_module(modulename, searchpath)
+        except ImportError:
+            return []
+        else:
+            if fd is not None:
+                fd.close()
+        if type_[2] != imp.PKG_DIRECTORY:
+            return [pathname]
+        else:
+            init_file = os.path.join(pathname, '__init__.py')
+            if os.path.getsize(init_file) == 0:
```

Comment:
Could you please put a comment explaining why do you check for the file size at this point?